File: frontend/src/api/models/UnitAssociationCodeResponseDTO.ts
export type UnitAssociationCodeResponseDTO = {
  /**
   * @description Código de associação
   * @type string
   */
  code: string
  /**
   * @description Data e hora de expiração do código
   * @type string, date-time
   */
  expiresAt: string
}

File: backend/src/main/java/br/com/visitsafe/dto/unit/UnitAssociationCodeResponseDTO.java
package br.com.visitsafe.dto.unit;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.OffsetDateTime;

@Data
@AllArgsConstructor
public class UnitAssociationCodeResponseDTO {

    @Schema(description = "Código de associação", required = true, example = "AB12CD34")
    private String code;

    @Schema(description = "Data e hora de expiração do código", required = true, example = "2025-05-12T18:00:00Z")
    private OffsetDateTime expiresAt;
}
